core/server/index.js

```diff
@@ -48,6 +49,7 @@ function initialiseServices() {
         mega.listen(),
         webhooks.listen(),
         appService.init(),
+        stats.init(),
```

Comment:
Apart from this place, we'd also need to hook into other model specific events (like adding/removing a member or post) to keep stat numbers fresh.

Reply:
Inside `updateGlobalTemplateOptions()` function, there is a todo like this:

https://github.com/TryGhost/Ghost/blob/b34f2e86fb9a785e7483f6ffc4a1b0269f9f6709/core/frontend/services/themes/middleware.js#L100

When I first wrote this code, I thought solving this problem would make this PR too big. But it seems that it wouldn't do so. I'll handle this.